test/apps/carto-trajectories/config.ts

```diff
@@ -0,0 +1,123 @@
+import {trajectoryQuerySource, trajectoryTableSource} from '@carto/api-client';
+
+// CARTO API configuration
+const API_CONFIG = {
+  apiBaseUrl: 'https://gcp-us-east1-19.dev.api.carto.com',
```

Comment:
Not urgent but would be great if we could include a demo dataset in a production env when merging this!